python/lsst/validate/drp/validate.py

```diff
@@ -200,9 +206,25 @@ def run(repo, visitDataIds, good_mag_limit=21.0,
         Expected photometric RMS [mmag] across visits.
     matchRef : int, optional
         Expectation of the number of stars that should be matched across visits.
+    makePrint : bool, optional
+        Print calculated quantities (to stdout).
+    makePlot : bool, optional
+        Create plots for metrics.  Saved to current working directory.
+    makeJson : bool, optional
+        Create JSON output file for metrics.  Saved to current working directory.
+    outputPrefix : str
+        Specify the beginning filename for output files.
+
+    Outputs
+    -------
+    Names of plot files or JSON file are generated based on repository name,
+    unless overriden by specifying `plotBase`.
+    E.g., Analyzing a repository "CFHT/output"
+        will result in filenames that start with "CFHT_output_".
     """
 
-    plotbase = repoNameToPrefix(repo)
+    if outputPrefix is None:
+        outputPrefix = repoNameToPrefix(repo)
```

Comment:
I finally realize that `plotBase` is not a path prefix but is part of the file name itself. Some of my earlier comments were based on that misunderstanding. Hopefully that shows you that documentation of `plotBase` will be useful. :smile: